MonoGame.Framework/Platform/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -187,6 +187,7 @@ protected virtual void CreateDeviceResources()
                 creationFlags |= SharpDX.Direct3D11.DeviceCreationFlags.Debug;
             }
 
+
```

Comment:
Looks like you accidentally added a line here.

Reply:
Yep, cut paste code to somewhere else left a trail behind :)